tests/flows/test_fpgaflow.py

```diff
@@ -522,3 +522,49 @@ def test_fpga_syn_extract(top_module,
         assert expected_macro_count == 1, \
             f'Expected one instance of {expected_macro},' \
             ' got {expected_macro_count} instances'
+
+
+@pytest.mark.quick
+def test_vpr_gen_post_implementation_netlist():
+    chip = Chip('foo')
+    chip.input('test.v')
+
+    part_name = 'faux'
+
+    # Create FPGA
+    fpga = FPGA(part_name)
+
+    fpga.set('fpga', part_name, 'var', 'vpr_device_code', 'faux')
+    fpga.set('fpga', part_name, 'var', 'vpr_clock_model', 'ideal')
+
+    with open('test.file', 'w') as f:
+        f.write('test')
+
+    fpga.set('fpga', part_name, 'file', 'archfile', 'test.file')
+    fpga.set('fpga', part_name, 'file', 'graphfile', 'test.file')
+
+    fpga.set('fpga', part_name, 'var', 'channelwidth', 50)
+
+    chip.use(fpga)
+    chip.set('fpga', 'partname', 'faux')
+
+    # 3. Load flow
+    chip.use(fpgaflow, fpgaflow_type='vpr')
+    chip.set('option', 'flow', 'fpgaflow')
+
+    chip.set('tool', 'vpr', 'task', 'route', 'var', 'gen_post_implementation_netlist', 'true')
```

Comment:
`'true'` -> `True`
while the value is stored as a string, that will be changed soon-ish so might as well get the setting side of things right